docs/src/modules/components/AdDisplay.js

```diff
@@ -34,7 +34,7 @@ export default function AdDisplay(props) {
     classes = autoShapeStyles();
   }
 
-  /* eslint-disable material-ui/no-hardcoded-labels, react/no-danger */
```

Comment:
Why was this removed?

Reply:
The rule has issues with the TypeScript parser. In this case hello is correctly identified as a *Literal*

```jsx
<div>{'hello'}</div>
```

In this case

```jsx
<div>hello</div>
```

it's not.

Reply:
Then we should fix the rule.

Reply:
I couldn't find any solution (I have looked). I would propose we disable the rule, for now.

Reply:
Then we can't apply the change. I don't see any benefit in removing so many safe guards, adding so much churn to the codebase to lint a  handful of TS files. It's obvious that `@typescript-eslint` is not ready yet to replace `babel-eslint` unless you want to put in more work.